frontend/src/components/pages/ReviewPage.tsx

```diff
@@ -42,6 +42,14 @@ const ReviewPage = () => {
   const [title, setTitle] = useState<string>("");
   const [language, setLanguage] = useState<string>("");
 
+  const [isCommenting, setIsCommenting] = useState(false);
+  const [commentLine, setCommentLine] = useState(0);
+
+  const [
```

Comment:
this state name is a little ambiguous - can it be `commentStoryTranslationContentId` or something similar?